docs/source/markdown/options/uidmap.container.md

```diff
@@ -2,16 +2,22 @@
 ####>   podman create, run
 ####> If file is edited, make sure the changes
 ####> are applicable to all of those.
-#### **--uidmap**=*container_uid:from_uid:amount*
+#### **--uidmap**=*[flags]container_uid:from_uid[:amount]*
 
 Run the container in a new user namespace using the supplied UID mapping. This
 option conflicts with the **--userns** and **--subuidname** options. This
 option provides a way to map host UIDs to container UIDs. It can be passed
 several times to map different ranges.
 
+The *amount* value is optional and it is assumed to be **1** if not given.
```

Comment:
I think you need to talk about flags too here, at least briefly. Perhaps?
```suggestion
The possible values of the optional *flags* are discussed further down on this page.
The *amount* value is optional and assumed to be **1** if not given.
```

and I'm not married to that suggestion if you have a better thought.